tests/bsim/bluetooth/mesh/src/test_suspend.c

```diff
@@ -165,6 +171,135 @@ struct bt_mesh_cfg_cli_mod_pub pub_params = {
 	.transmit = 0,
 };
 
+extern const struct bt_mesh_comp comp;
+/* For legacy adv, pb-gatt advs are sent with a 1000ms interval. For ext adv, they are sent
+ * with a 100ms interval.
+ */
+static struct bt_mesh_test_gatt gatt_param = {
+#if defined(CONFIG_BT_EXT_ADV)
+	/* (total transmit duration) / (transmit interval) */
+	.transmits = 1500 / 100,
+	.interval = 100,
+#else
+	.transmits = 2000 / 1000,
+	.interval = 1000,
+#endif
+	.service = MESH_SERVICE_PROVISIONING,
+};
+
+static bool gatt_check_rx_count(uint8_t transmit)
+{
+	static int cnt;
+
+	LOG_INF("rx: cnt(%d)", cnt);
```

Comment:
In general I can appreciate adding more log to the Bsim tests 👍 But I think most of the `LOG_INF` can be converted to `LOG_DBG`. For most iterations of this test most of this log is not relevant. If anybody needs to take a closer look they can modify `LOG_MODULE_REGISTER(test_suspend, LOG_LEVEL_INF);` to enable this logging.